src/sentry/models/orgauthtoken.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import annotations
+
+from django.db import models
+from django.utils import timezone
+from django.utils.encoding import force_text
+
+from sentry.db.models import (
+    ArrayField,
+    BaseManager,
+    FlexibleForeignKey,
+    Model,
+    control_silo_only_model,
+    sane_repr,
+)
+from sentry.models.orgauthtokenproject import OrgAuthTokenProject
+
+
+@control_silo_only_model
+class OrgAuthToken(Model):
+    __include_in_export__ = True
+
+    organization = FlexibleForeignKey("sentry.Organization", null=False)
+    projects = models.ManyToManyField(
+        "sentry.Project", related_name="projects", through=OrgAuthTokenProject
+    )
+    # max JWT token length is 2048, +7 for our custom prefix (sntrys_)
+    token = models.CharField(max_length=2055, unique=True, null=False)
+    label = models.CharField(max_length=255, null=False)
+    scope_list = ArrayField(of=models.TextField)
+
+    user_added = FlexibleForeignKey("sentry.User", null=True)
```

Comment:
`user_added` refers to the `User` that created this token? If so, I would call this `user` or `added_by`.

Reply:
yes, that's the idea - I'll rename it to `added_by` (IMHO `user` is a potentially confusing as it may imply this token _belongs_ to that user, which it doesn't).

Reply:
Oh yeah, that's a good point!